grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/query/ReasonerQueryImpl.java

```diff
@@ -360,17 +360,6 @@ public GetQuery getQuery() {
         return varTypeMap;
     }
 
-    @Override
-    public ImmutableMap<Var, Type> getVarTypeMap(Answer sub) {
-        return ImmutableMap.copyOf(getVarTypeMap(
-                Stream.concat(
-                        getAtoms(IsaAtom.class),
-                        inferEntityTypes()
-                )
-                )
-        );
-    }
-
```

Comment:
ok it probably should be `inferEntityTypes(sub)` here so that the sub is actually used in all the infer* methods

Reply:
But it wasn't in use though? I can add it back but nothing used it.

Reply:
`getVarTypeMap(sub`) is referenced in the infer functions in `RelationshipAtom` and others

Reply:
Yes I saw that, but `sub` was never used so I removed the parameter. At that stage I noticed that `public ImmutableMap<Var, Type> getVarTypeMap()` and `public ImmutableMap<Var, Type> getVarTypeMap(Answer sub) ` pretty much did the same thing.

So now everything calls `getVarTypeMap()` and it all still appears to be working. 

How are these methods different from each other that it's worth keeping both?

Reply:
the one with sub infers types from instance ids